### PR TITLE
TE-962: Apply the Container to the Header & Footer

### PR DESCRIPTION
**Pull request by stephenkr**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-962)

### What **one** thing does this PR do?
Applies the `Container` to the `Header` and `Footer`

### Any other notes
- I had to slightly refactor the `Footer` to accommodate the `Container`, I added two divs with the classes `.bottom-navigation` and `.top-navigation` to contain the `Menu` components
- The content of the `Header` and `Footer` will now be restricted to `1040px`

### Footer

__Before__
screen shot 2018-09-03 at 16 04 02

__After__
screen shot 2018-09-03 at 16 03 43

### Header

__Before__
screen shot 2018-09-03 at 16 17 31

__After__
screen shot 2018-09-03 at 16 17 14

### CODE REVIEW

File: src/components/collections/Header/component.js [hunk 1]
```diff
@@ -15,14 +16,16 @@ import { getDesktopMenuMarkup } from './utils/getDesktopMenuMarkup';
...
-  <Menu borderless className="is-header" text>
-    {getLogoMarkup(props.logoSrc, props.logoText)}
-    <Menu.Menu position="right">
-      {props.isUserOnMobile
-        ? getMobileMenuMarkup(props)
-        : getDesktopMenuMarkup(props)}
-    </Menu.Menu>
-  </Menu>
+  <div className="is-header">
+    <Container as={Menu} borderless className="is-header" text>
```
Comment (marcdavi-es) [inline src/components/collections/Header/component.js]: 1. Do we need the `div` wrapper?
1. Here we have `is-header` on the parent and child elements. Is that intentional?

Comment (stephenkr) [reply src/components/collections/Header/component.js]: 1. I think so, its good to have an overall wrapper that may in the future have a background colour if we chose.
2. I'll remove the class and just set the element to be `header`

File: src/styles/semantic/themes/livingstone/collections/menu.overrides [hunk 2]
```diff
@@ -342,3 +283,78 @@
 /* Resize large sizes */
 
 /* Sizes */
+
+/* Footer */
+.is-footer {
+
+  .top-navigation {
```
[the remark this answers is not in the document]
Comment (stephenkr) [reply src/styles/semantic/themes/livingstone/collections/menu.overrides]: These are used to contain the two menus, the classes were added to better specify which menu the css should be applied too.

I have also applied a background color to the two elements to have the background stretch full width.

[the remark this answers is not in the document]
Comment (marcdavi-es) [reply src/styles/semantic/themes/livingstone/collections/menu.overrides]: 1. Are the existing children selectors not specific enough? 
2. So we have two layers of background color, one on the containing div and one on the child menu? I'm not sure that's good.

**Comment by stephenkr** [comment]
@marcdavi-es Sorry on your comment above, there is no reply button.

Nope we cannot do this, because the menu CSS expects the item to appear directly after the menu component

**Comment by stephenkr** [comment]
Thanks @marcdavi-es, that sounds good, i'll apply your suggestions. :)

**Pull request merged**